apps/examples/smartfs/buffer_optimization/buffer_optimization_main.c

```diff
@@ -276,7 +276,7 @@ int buffer_optimization_main(int argc, char *argv[])
 			file_size = atoi(argv[2]);
 		}
 	} else {
-		printf("\nCorrect test command usage is: smartfs_buf_opt <Timer Device No.> <File Size in Kb>\n");
+		printf("\nCorrect test command usage is: jsmartfs_buf_opt <Timer Device No.> <File Size in Kb>\n");
```

Comment:
Since you have changed this
the app name in the Makefile needs to be changed too so that the test can be started with this command name
Please change, build, test and confirm.